srv/components/ha/cortx-ha/config.sls

```diff
@@ -44,5 +38,3 @@ No HA config on secondary node:
 Run cortx-ha init:
   cmd.run:
     - name: __slot__:salt:setup_conf.conf_cmd('/opt/seagate/cortx/ha/conf/setup.yaml', 'ha:init')
-    - require:
```

Comment:
Let's maintain these require statements to ensure installation.
Any specific reason this is removed here?

Reply:
please check the above response